src/Symfony/Component/Lock/Store/SemaphoreStore.php

```diff
@@ -37,7 +37,7 @@ public static function isSupported($blocking = null)
             return false;
         }
 
-        if ($blocking === false && \PHP_VERSION_ID < 50601) {
```

Comment:
the removed check is always "false", to the "&&" is false also, so the full "if" block can be removed

Reply:
Right. Block removed.

Reply:
in master, the removed check is always true

Reply:
nvm